### PR TITLE
Add support for compose-image-rtt-tests in pdc client

### PR DESCRIPTION
**Pull request by ycheng-aa**
Support list, info, update sub-commands.
Only test result is allowed to update.

JIRA: PDC-1260

### CODE REVIEW

**Comment by ycheng-aa** [comment]
@simozhan  @bliuredhat Please help me review the pull request. Thanks.

File: tests/compose_image_rtt_tests/tests.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Copyright (c) 2015 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+from pdc_client.test_helpers import CLITestCase
+from pdc_client.runner import Runner
+from copy import deepcopy
+
+
+class BuildImageTestCase(CLITestCase):
```
Comment (simozhan) [inline tests/compose_image_rtt_tests/tests.py]: this name should be changed, right?

Comment (ycheng-aa) [inline tests/compose_image_rtt_tests/tests.py]: Thanks for the careful review. Fixed in a3c4e23cabbd4a46.

**Comment by bliuredhat** [comment]
:+1:

File: pdc_client/plugins/compose_image_rtt_tests.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        list_parser.set_defaults(func=self.list_compose_image_rtt_tests)
+
+        info_parser = self.add_action('info', help='display details of a compose image RTT test')
+        self._add_common_arguments(info_parser)
+        info_parser.set_defaults(func=self.compose_image_rtt_test_info)
+
+        help_text = 'update an existing compose image\'s RTT test result (untested, passed or failed)'
+        update_parser = self.add_action('update', help=help_text)
+        self._add_common_arguments(update_parser)
+        update_parser.add_argument('test_result', metavar='TEST_RESULT')
```
Comment (bliuredhat) [inline pdc_client/plugins/compose_image_rtt_tests.py]: I think this format is not perfect, you can check other api. For example:
usage: pdc compose-image-rtt-tests update [-h]  [--test_result  TEST_RESULT]
                                          COMPOSE_ID VARIANT_UID ARCH   FILE_NAME

Comment (ycheng-aa) [inline pdc_client/plugins/compose_image_rtt_tests.py]: I think your suggestion is more clear. Fixed in eefb0c6391ebdc. Thanks.

Now it looks like:
usage: pdc compose-image-rtt-tests update [-h] [--test-result TEST_RESULT]
                                          COMPOSE_ID VARIANT_UID ARCH
                                          FILE_NAME

**Comment by bliuredhat** [comment]
:+1:

**Pull request merged**